src/components/Section/Citizenship/Citizenship.jsx

```diff
@@ -1,198 +1,98 @@
 import React from 'react'
+import { Route } from 'react-router'
 import { connect } from 'react-redux'
 import { i18n } from '@config'
-import SectionElement from '@components/Section/SectionElement'
-import { SectionViews, SectionView } from '@components/Section/SectionView'
-import { Field } from '@components/Form'
-import Status from '@components/Section/Citizenship/Status'
-import Multiple from '@components/Section/Citizenship/Multiple'
-import Passports from '@components/Section/Citizenship/Multiple/Passports'
+import { ErrorList } from '@components/ErrorList'
+import SectionNavigation from '@components/Section/shared/SectionNavigation'
+import { CITIZENSHIP } from '@constants/sections'
+import * as formTypes from '@config/formTypes'
+import Intro from './Intro'
+import Status from './Status'
+import Multiple from './Multiple'
+import Passports from './Multiple/Passports'
+import Review from './Review'
 
-class Citizenship extends SectionElement {
-  render() {
-    return (
-      <div>
-        <SectionViews
-          current={this.props.subsection}
-          dispatch={this.props.dispatch}
-          update={this.props.update}>
+class Citizenship extends React.Component {
+  constructor(props) {
+    super(props)
 
-          <SectionView
-            name="intro"
-            back="relationships/review"
-            backLabel={i18n.t('relationships.destination.review')}
-            next="citizenship/status"
-            nextLabel={i18n.t('citizenship.destination.status')}>
-            <div>
-              <h1 className="section-header">{i18n.t('citizenship.intro.title')}</h1>
-              <Field
-                optional={true}
-                className="no-margin-bottom">
-                {i18n.m('citizenship.intro.body')}
-              </Field>
-            </div>
-          </SectionView>
+    this.form = formTypes[props.formType]
+    this.section = this.form.find(section => (section.key === CITIZENSHIP))
+
+    // Each key in subsectionLibrary corresponds to the subsection
+    // name in the config file.
+    this.subsectionLibrary = {
```

Comment:
There are many subsections that are specific to a certain form. Because the Citizenship component will be shared amongst forms, I created this `subsectionLibrary` that has all the possible subsections. The `key` is the `name` attribute in the subsection config object and the `value` is the subsection component.